cmake/modules/RootCPack.cmake

```diff
@@ -29,12 +29,26 @@ string(REGEX REPLACE "^([0-9]+).*$" "\\1" CXX_MAJOR ${CMAKE_CXX_COMPILER_VERSION
 string(REGEX REPLACE "^([0-9]+)\\.([0-9]+).*$" "\\2" CXX_MINOR ${CMAKE_CXX_COMPILER_VERSION})
 
 #---Resource Files-----------------------------------------------------------------------------------
-configure_file(README.md README.md COPYONLY)
 configure_file(LICENSE LICENSE.txt COPYONLY)
 configure_file(LGPL2_1.txt LGPL2_1.txt COPYONLY)
-set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_BINARY_DIR}/README.md")
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_BINARY_DIR}/LICENSE.txt")
-set(CPACK_RESOURCE_FILE_README "${CMAKE_BINARY_DIR}/README.md")
+if (APPLE)
+  find_program(CONVERTER textutil)
```

Comment:
Wouldn't it be easier to just rename it to a plain text file instead of depending on an external tool to convert to HTML?

Reply:
That's for installers - and if we rename the file they will display "raw" markdown. Displaying HTML is something that all installers supported by CPack can do, apparently.

Reply:
If `textutil` comes with Mac OS, then converting to HTML is a no-brainer. If not, an alternative to requiring the external tool is to convert to HTML and commit the result.